contracts/src/plugin/extensions/proposal/Proposal.sol

```diff
@@ -11,9 +11,19 @@ import {IProposal} from "./IProposal.sol";
 /// @notice An abstract contract containing the traits and internal functionality to create and execute proposals that can be inherited by non-upgradeable DAO plugins.
 /// @custom:security-contact sirt@aragon.org
 abstract contract Proposal is IProposal, ERC165 {
+    error FunctionDeprecated();
```

Comment:
- [ ] Explain why this function is deprecated here
- [x] Explain it also on the natspec of `proposalCount()` on `IProposal`